.gitignore

```diff
@@ -7,6 +7,7 @@
 coverage*
 **/.DS_Store
 **/main
+cmd/docx-tool/docx-tool
```

Comment:
I think we have defined the tools in the Makefile, so look at that and there it will be built into an already existing binary folder.

Reply:
Where do you see a Makefile in this repo? I didn't find one, so I went to the cmd/docx-tool directory and ran "go build" which created this file.

Reply:
Oh, sorry. Thought it was the kvasir one.